### ISSUE TITLE
Hide Dashboard During Intro Sequence

### ISSUE THREAD

**Issue by skybaks** [codecat/tm-dashboard#28]
Dashboard is visible during a map's intro sequence. It would be nice if it was hidden since it wont be showing any information during that time.

**Comment by codecat** [comment]
Can we detect if the intro mediatracker is playing? I know there are some instances of the mediatracker player around, perhaps those can be queried to see if the intro is being played.

**Comment by skybaks** [comment]
I think there are some state variables in the UIMgr that might indicate if we are in an intro. As you say, it would take some investigating to find whether it works consistently in all cases.

**Comment by codecat** [comment]
The `EUISequence` enum in [`CGamePlaygroundUIConfig`](https://next.openplanet.nl/Game/CGamePlaygroundUIConfig) looks promising.

**Comment by codecat** [comment]
This setting apparently breaks watching replays in solo, see #35. Could we do something about this? It seems like the sequency type is set to EndRound both during the intro and while watching a ghost.

**Comment by skybaks** [comment]
Yeah, it did cross my mind that 'not Playing' would be too far sweeping of logic. I can investigate UI sequences further.
